src/smart-components/Message/utils/tokens/tokenize.ts

```diff
@@ -0,0 +1,105 @@
+import { User } from "@sendbird/chat";
+import { USER_MENTION_PREFIX } from "../../consts";
+import { IdentifyMentionsType, Token, TOKEN_TYPES, TokenParams } from "./types";
+import { isUrl } from "../../../../utils";
+
+export function getUserMentionRegex(mentionedUsers: User[], templatePrefix_: string): RegExp {
+  const templatePrefix = templatePrefix_ || USER_MENTION_PREFIX;
+  return RegExp(`(${mentionedUsers.map(u => `${templatePrefix}{${u.userId}}`).join('|')})`, 'g');
+}
+
+export function identifyMentions({
```

Comment:
```suggestion
function identifyMentions({
```
Seems it's used only in this file. Will it be useful if we use this kind of deadcode detector? 
[ts-prune](https://blog.logrocket.com/how-detect-dead-code-frontend-project/#using-typescript-to-detect-dead-code)

Reply:
exporting to add test

Reply:
About ts-prune, if you want, we can try